src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.container.js

```diff
@@ -15,14 +15,14 @@ import { useHideWidgetByRuleLocations } from '../Enrollment/EnrollmentPageDefaul
 import { buildUrlQueryString, useLocationQuery } from '../../../utils/routing';
 import { deleteEnrollment } from '../Enrollment/EnrollmentPage.actions';
 import { buildEnrollmentsAsOptions } from '../../ScopeSelector';
-import { convertValue } from '../../../converters/clientToView';
+import { convertDateWithTimeForView, convertValue } from '../../../converters/clientToView';
```

Comment:
NIT: export and import `convertDateWithTimeForView` from [converters/index.js ](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/converters/index.js)